charts/agent-k8s/values.yaml

```diff
@@ -59,6 +59,10 @@ fullnameOverride: ""
 controllerNodeSelector: {}
 # -- Kubernetes Node Selector for assigning worker agents and scheduling agent tasks to specific nodes in the cluster.
 # The selector must match a node's labels for the pod to be scheduled on that node.
+controllerTolerations: {}
+# -- Kubernetes Node Tolerations for the controller.
+# Expects input structure as per specification <https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#toleration-v1-core>.
+# Example: `--set controllerTolerations[0].operator=Equal,controllerTolerations[0].effect=NoSchedule,controllerTolerations[0].key=dedicated,controllerTolerations[0].value=scalr-agent-worker-pool`
```

Comment:
@yurii-kryvosheia there is a copypasted value from 'workerTolerations' that might be confusing as an example  - `value=scalr-agent-worker-pool`. It would be better to change it to `value=scalr-agent-controller-pool`.